src/components/table/TableCellEdit.vue

```diff
@@ -1,10 +1,12 @@
 <template>
   <div class="table-cell-editable">
+    <!-- Edge does not respect lang="en-US" as such to not have ',' as decimal separator we need type="text" -->
     <input
       :ref="`${props.item.date}-${props.id}-value`"
       v-model.number="currentItem.y"
       class="table-cell-edit"
-      type="number"
+      type="text"
+      inputmode="decimal"
```

Comment:
Edge does respect `inputmode`?

Reply:
it respects the fact that it won't modify a string passed with a `.` to a `,` if the `type="text"`